docs/source/peers/peers.md

```diff
@@ -361,74 +356,40 @@ proposal P generating transaction T1 response R2 which it endorses with E2.
-comparison before non-determinism can be detected. (Strictly speaking, even this
-is not enough, but we defer this discussion to the transaction section, where
-non-determinism is discussed in detail.)
-
-At the end of phase 1, the application is free to discard inconsistent
-transaction responses if it wishes to do so, effectively terminating the
-transaction workflow early. We'll see later that if an application tries to use
-an inconsistent set of transaction responses to update the ledger, it will be
-rejected.
-
-### Phase 2: Ordering and packaging transactions into blocks
-
-The second phase of the transaction workflow is the packaging phase. The orderer
-is pivotal to this process --- it receives transactions containing endorsed
-transaction proposal responses from many applications, and orders the
-transactions into blocks. For more details about the
-ordering and packaging phase, check out our
-[conceptual information about the ordering phase](../orderer/ordering_service.html#phase-two-ordering-and-packaging-transactions-into-blocks).
-
-### Phase 3: Validation and commit
-
-At the end of phase 2, we see that orderers have been responsible for the simple
-but vital processes of collecting proposed transaction updates, ordering them,
-and packaging them into blocks, ready for distribution to the peers.
-
-The final phase of the transaction workflow involves the distribution and
-subsequent validation of blocks from the orderer to the peers, where they can be
-committed to the ledger. Specifically, at each peer, every transaction within a
-block is validated to ensure that it has been consistently endorsed by all
-relevant organizations before it is committed to the ledger. Failed transactions
-are retained for audit, but are not committed to the ledger.
+-->
+
+Note that a peer endorses a proposal response by adding its digital signature, and
+signing the entire payload using its private key. This endorsement can be subsequently
+used to prove that this organization's peer generated a particular response.
+
+### Phase 2: Transaction Ordering
+
+The second phase of the transaction workflow is the ordering and packaging phase.
+The ordering service (running on orderer nodes) receives transactions containing signed and
+endorsed proposal responses, from one or more applications via the gateway service, and
+orders and packages the transactions into blocks. These are the blocks (which are also ordered) --- consisting
+of validated, endorsed and ordered transactions --- that make up a Fabric blockchain ledger.
+
+For details about the ordering and packaging phase, see
+the [conceptual information about the ordering phase](../orderer/ordering_service.html#phase-two-ordering-and-packaging-transactions-into-blocks).
+
+### Phase 3: Validation and Commitment
+
+The third and final phase of the transaction workflow is the distribution of ordered
+transactions from orderers to peers (i.e. from the ordering service to the gateway
```

Comment:
Ordering service does not distribute blocks to gateway service.
Ordering service distributes blocks to all peers.
Gateway service will listen for peer's transaction commit event before responding back to client.

Reply:
ok.